docs/components/utilities/package.md

```diff
@@ -1,4 +1,139 @@
+
+![Version Name](/assets/images/components/utilities/package/p_version-name.png)
+
+_**\(** Getter only **\)**_
+
+`Type: Alphanumerical`
+
+Returns the version name of the currrent running application.
+
+## Methods
+
+### App Name From
+
+![App Name From](/assets/images/components/user-interface/utilities/package/m_app-name-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the name of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
+
+### Get Package Icon
+
+![Get Package Icon](/assets/images/components/user-interface/utilities/package/m_get-package-icon.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the icon of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
+
+### Is Package Installed
+
+![Is Package Installed](/assets/images/components/user-interface/utilities/package/m_is-package-installed.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns true if the package (app) is installed.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
+
+### Version Code From
+
+![Version Code From](/assets/images/components/user-interface/utilities/package/m_version-code-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the version code of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
+
+### Version Name From
+
+![Version Name From](/assets/images/components/user-interface/utilities/package/m_version-name-from.png)
+
+_**\(** Text `package Name` **\)**_
+
+Returns the version name of the app from the given package name.
+
+Params               |  []()       
+-------------------- | ------- 
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
+
+### Is Permission Granted
+
+![Is Permission Granted](/assets/images/components/user-interface/utilities/package/m_is-permission-granted.png)
+
+_**\(** Text `permission Name`, Text `package Name` **\)**_
+
+Checks whether a particular package (app) has been granted a particular permission.
+
+Params               |  []()       
+-------------------- | ------- 
+`permission Name`    | **Text:** The name of the permision that has to be checked. For example, READ_CALENDER.
+`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, io.kodular.myfirstapp
```

Comment:
```suggestion
`package Name`       | **Text:**  The package name of the app whose name has to be returned. For example, _io.kodular.myfirstapp_.
```

Reply:
Ah! This branch has some problems. Closing this pull request for now, will open a new one.